### PR TITLE
fix double period issue

### PR DESCRIPTION
**Pull request by orlichter1**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/27229

## Description
in SMTP mails lines sometimes start with `..` instead of `.` due to SMTP standard. This wasn't processed correctly and is fixed now.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py [hunk 1]
```diff
@@ -3432,6 +3432,7 @@ def handle_eml(file_path, b64=False, file_name=None, parse_only_headers=False, m
             # decode bytes taking into account BOM and re-encode to utf-8
             file_data = file_data.decode("utf-8-sig").encode("utf-8")
 
+        file_data = file_data.replace('\r\n..', '\r\n.')
```
Comment (yaakovi) [inline Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py:3435]: will there always be `\r`?

File: Packs/CommonScripts/pack_metadata.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "name": "Common Scripts",
     "description": "Frequently used scripts pack.",
     "support": "xsoar",
-    "currentVersion": "1.2.36",
+    "currentVersion": "1.3.0",
```
Comment (yaakovi) [inline Packs/CommonScripts/pack_metadata.json:5]: should be `1.2.37`. this is a bug fix, not an enhancement

**Review by yaakovi — APPROVED** [review]

File: Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py (hunk 1 above)
Comment (orlichter1) [reply Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py:3435]: On the few examples I found `\r` apeared

File: Packs/CommonScripts/pack_metadata.json (hunk 2 above)
Comment (orlichter1) [reply Packs/CommonScripts/pack_metadata.json:5]: Done automaticaly by the sdk (did `-u minor`) I'll fix it

File: Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py [hunk 3]
```diff
@@ -3745,4 +3749,4 @@ def main():
 
 
 if __name__ in ('__builtin__', '__main__'):
-    main()
+    handle_eml('/Users/olichter/dev/demisto/content/Packs/CommonScripts/Scripts/ParseEmailFiles/test_data/multiple_to_cc.eml')
```
Comment (bakatzir) [inline Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py:3752]: I think this is by mistake

Comment (orlichter1) [reply Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py]: Good catch :)

**Comment by kirbles19** [comment]
@orlichter1 Done.

**Pull request merged**